src/Makemodule-local.am

```diff
@@ -182,7 +187,8 @@ install-exec-hook:
 	mkdir -p $(DESTDIR)$(includedir)/$(PACKAGE) && \
 	    cd $(DESTDIR)$(includedir) && \
 	    mv -f `ls -1A | egrep -v '^'"$(PACKAGE)"'$$' ` $(PACKAGE)/
-	mkdir -p "$(DESTDIR)$(prefix)/libexec/$(PACKAGE)"
+	mkdir -p "$(DESTDIR)$(prefix)/libexec/$(PACKAGE)" && \
```

Comment:
The `&&` here is superfluous : `make` in any case would fail if a sub-shell script fails (a `make` line including concatenation of tails). So an `mkdir` failing would not let `mv` happen.

The danger here is maintainability and attention: if/when we change paths for this binary, we'd change the mkdir (they are together, right?) and then the next command for helpers will have no dir to work with.